dapla-manual/statistikkere/datadoc.qmd

```diff
@@ -1,28 +1,16 @@
 # Datadoc
 
-::: {.callout-warning}
-NB! Advarsel - Ustabilitet i Dapla Beta!
-:::
-
-Statistikkmetadata er strukturert informasjon om statistikk. Dette omfatter informasjon som brukes for å produsere, formidle, forstå, finne
-eller gjenbruke statistikk. For å kunne gjenfinne data i SSB er man helt avhengig av at det finnes et enhetlig system for metadata knyttet til dataene.
-
-## Hva er Datadoc
-
-Datadoc er en tjeneste som skal forenkle arbeidet med å dokumentere datasett og variabelforekomster.
-Felt og datatyper er definert som JSON schema i [ssb-datadoc-model](https://github.com/statisticsnorway/ssb-datadoc-model).
-
-Datadoc har et brukergrensesnitt som kan nåes fra Dapla Lab.
-
-Foreløpig støtter løsningen følgende filformater:
+Datadoc er et verktøy for å dokumentere datasett og variablene som utgjør datasett. Metadataen som er skapt kommer til å bli selve ryggmargen til SSBs fremtidig datakatalog og skal gjør det mulig for SSB ansatt og det norske samfunnet å finne fram til faktagrunnlaget.
```

Comment:
"...utgjør datasett**et**. Metadataen**e** som **skapes** kommer til å bli selve ryggmargen til SSBs fremtidig**e** datakatalog og skal gjør**e** det mulig for SSB ansatt**e** og det norske samfunnet å finne fram til faktagrunnlaget.